src/ClockPanel.tsx

```diff
@@ -215,3 +206,16 @@ export class ClockPanel extends PureComponent<Props, State> {
     );
   }
 }
+
+export const ClockPanel = withTheme2(UnthemedClockPanel);
+
+const getStyles = stylesFactory((theme: GrafanaTheme2, bgColor: string | undefined) => ({
+  wrapper: css`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    background-color: ${bgColor ? theme.visualization.getColorByName(bgColor) : null};
```

Comment:
Is this the only real change?  can we keep 7.3 dependencies and use something like:
 
```suggestion
    background-color: ${bgColor ? getColorForTheme(theme, bgColor) : null};
```